README.md

```diff
@@ -182,6 +173,32 @@ decoder = PerceiverDecoder(
 mnist_classifier = PerceiverIO(encoder, decoder)
 ```
 
+## Development
+
+### Update project dependencies in conda environment
```

Comment:
Replace all level-3 headlines in section `Development` with plain sentences and an `:` at the end?